addons/popochiu/engine/objects/region/popochiu_region.gd

```diff
@@ -121,48 +126,71 @@ func _check_scaling(
 	area_rid: RID, area: Area2D, area_shape_index: int, local_shape_index: int, entered: bool
 ):
 	if not (
-		area is PopochiuCharacter 
-		and area.get_node_or_null("ScalingPolygon") 
-		and area_shape_index == area.get_node("ScalingPolygon").get_index()
+		area is PopochiuCharacter
+		and area.get("scaling_polygon")
```

Comment:
Is it worth to have a null-safe check here?

Reply:
You mean for the `area`?

Reply:
Yes, exactly!